pyoxidizer/src/py_packaging/distutils.rs

```diff
@@ -46,70 +46,26 @@ lazy_static! {
 /// modified distutils will survive multiple process invocations, unlike a
 /// monkeypatch. People do weird things in setup.py scripts and we want to
 /// support as many as possible.
-pub fn prepare_hacked_distutils(
-    logger: &slog::Logger,
-    dist: &ParsedPythonDistribution,
-    dest_dir: &Path,
-    extra_python_paths: &[&Path],
-) -> Result<HashMap<String, String>, String> {
-    let extra_sys_path = dest_dir.join("packages");
-
+pub fn prepare_hacked_distutils(logger: &slog::Logger, target: &PythonPaths) {
     warn!(
         logger,
         "installing modified distutils to {}",
-        extra_sys_path.display()
+        target.stdlib.display()
     );
 
-    let orig_distutils_path = dist.stdlib_path.join("distutils");
-    let dest_distutils_path = extra_sys_path.join("distutils");
-
-    for entry in walkdir::WalkDir::new(&orig_distutils_path) {
-        match entry {
-            Ok(entry) => {
-                if entry.path().is_dir() {
-                    continue;
-                }
-
-                let source_path = entry.path();
-                let rel_path = source_path
-                    .strip_prefix(&orig_distutils_path)
-                    .or_else(|_| Err("unable to strip prefix"))?;
-                let dest_path = dest_distutils_path.join(rel_path);
-
-                let dest_dir = dest_path.parent().unwrap();
-                std::fs::create_dir_all(&dest_dir).or_else(|e| Err(e.to_string()))?;
-                std::fs::copy(&source_path, &dest_path).or_else(|e| Err(e.to_string()))?;
-            }
-            Err(e) => return Err(e.to_string()),
-        }
-    }
+    let dest_distutils_path = target.stdlib.join("distutils");
 
     for (path, data) in MODIFIED_DISTUTILS_FILES.iter() {
         let dest_path = dest_distutils_path.join(path);
 
+        if !dest_path.exists() {
```

Comment:
This is probably why the Win jobs are green but failing.  This was all working this morning.
The log say "not updating missing distutils/command/build_ext.py" so a new bug has crept in.